src/lib/accessibility/Behaviors/Toolbar/ToolbarButtonBehavior.ts

```diff
@@ -0,0 +1,19 @@
+import { Accessibility } from '../../interfaces'
+
+const ToolbarButtonBehavior: Accessibility = (props: any) => ({
+  attributes: {
+    root: {
+      role: 'presentation',
+    },
+    anchor: {
+      role: 'button',
+      tabIndex: '0',
+      'aria-disabled': props['disabled'],
+      'aria-label': props['aria-label'],
+      'aria-labelledby': props['aria-labelledby'],
+    },
+  },
+  handledProps: ['aria-label', 'aria-labelledby'],
```

Comment:
`aria-disabled` should be declared as `handled` one as well, related change to `anchor`'s attribute value should be introduced:

```jsx
anchor: {
     ...
      'aria-disabled': props['disabled'] || props['aria-disabled'],
      ...
    },
```